### PR TITLE
Migrate plugin to the structure used since Shopware v5.2

### PR DESCRIPTION
**Pull request by philipgatzka**
[The documentation](https://developers.shopware.com/developers-guide/shopware-5-media-service/#existing-adapters) refers to this plugin and the other adapters from shopwareLabs already have been migrated.

### CODE REVIEW

File: SwagMediaSftp.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * (c) shopware AG <info@shopware.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace SwagMediaSftp;
+
+require __DIR__ . "/vendor/autoload.php";
```
Comment (mitelg) [inline SwagMediaSftp.php]: why is this necessary?

File: plugin.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<plugin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/plugin.xsd">
+
+    <label lang="de">Media Adapter: SFTP</label>
+    <label lang="en">Media Adapter: SFTP</label>
+
+    <version>1.0.0</version>
+    <author>shopware AG</author>
+    <license>MIT</license>
+    <description>SFTP-Erweiterung für die Media Adapter</description>
```
Comment (mitelg) [inline plugin.xml]: also translate the description

File: SwagMediaSftp.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+use Shopware\Components\Plugin;
+use Shopware\Components\Plugin\Context\ActivateContext;
+use Shopware\Components\Plugin\Context\DeactivateContext;
+use Shopware\Components\Plugin\Context\InstallContext;
+use Shopware\Components\Plugin\Context\UninstallContext;
+use Shopware\Components\Plugin\Context\UpdateContext;
+
+class SwagMediaSftp extends Plugin
+{
+    public function update(UpdateContext $context)
```
Comment (mitelg) [inline SwagMediaSftp.php]: the overwriting of these four methods is completely unnecessary, as they do exakt the same as the parent methods

File: SwagMediaSftp.php (hunk 1 above)
Comment (teiling88) [reply SwagMediaSftp.php]: needed for this lib "league/flysystem-sftp"

Comment (mitelg) [reply SwagMediaSftp.php]: is not used here. should be in the subscriber file, shouldn't it?

Comment (teiling88) [reply SwagMediaSftp.php]: I think it make no real difference but It could be a better place there :-)

File: SwagMediaSftp.php (hunk 3 above)
Comment (philipgatzka) [reply SwagMediaSftp.php]: Indeed, I didn't check that. Removed the methods

File: plugin.xml (hunk 2 above)
Comment (philipgatzka) [reply plugin.xml]: Done
